portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,54 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Kayla Leung Portfolio</title>
+    <title>Kayla Leung</title>
+
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
```

Comment:
Also, since you've removed the greeting container, this script is going to fail now so you might as well remove it.